### ISSUE TITLE
How to Access the OctoPi Terminal, without SSH, when OctoDash auto-starts.

### ISSUE THREAD

**Issue by metasean** [UnchartedBull/OctoDash#3038]
--- 

tldr: I'm feeling like an idiot, but how the heck do I access the OctoPi terminal, without SSH, when OctoDash is successfully auto-starting (which is normally a great thing), and thus taking control of all of the user interface?

---

I'm feeling like an idiot here, because I'm sure this is something super-simple and well-documented, but for the life of me I'm not seeing how to do it or finding documentation on how to do it. 😩 

I successfully installed OctoPi and OctoDash several months ago and, for the most part, I've been really enjoying them! 👍 

The one big problem that I have, is that the attached webcam's video is only showing an itty-bitty postage sized image, despite me having it connected to a 7 inch touchscreen.  As I understand it, I need to [update my config file](https://github.com/PrusaMK2Users/MK2_Tips_and_Tricks/wiki/OctoPrint-Webcam-Setup), which makes perfect sense. 👍 

I thought I had set-up SSH when I set-up OctoPi & OctoDash, but when I try to SSH in, I get a `port 22: Connection refused` error, so apparently I didn't set up SSH. 😞  

Next, I grabbed a USB keyboard and connected it to my OctoPi machine, but **_I have no idea how to get to the terminal now that OctoDash is working._** When I boot up, I still see OctoPi doing all of it's start-up procedures, but as soon as it's booted, OctoDash auto-starts, so I can't really access the terminal before OctoDash starts, and I can't figure out how to switch to, or open, the OctoPi's terminal once OctoDash is running, nor can I find anyway to stop OctoDash without also having OctoPi stop. 😫 

**_So, how the heck do I access the OctoPi terminal, without SSH, when OctoDash is successfully auto-starting and taking control of all of the user interface?_**

**Comment by jneilliii** [comment]
With keyboard you can use Ctrl+k and or Ctrl+ t. I think the skull icon in custom actions may also do it.
